stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -184,11 +183,35 @@ private func configureRuntimeFromEnvironment() {
       return
     }
 
-    guard address.prefix(7) == "grpc://" else {
-      fatalError("SWIFT_TENSORFLOW_SERVER_ADDRESS must start with 'grpc://'.")
+    if let idx = address.firstIndex(of: ":") {
+      if let endIdx = address.index(idx, offsetBy: 3, limitedBy:address.endIndex) {
+        if address[idx..<endIdx] == "://" {
+          let `protocol` = address[address.startIndex..<idx]
+          let target = address[endIdx..<address.endIndex]
+          _RuntimeConfig.session = .remote(serverDef: """
+cluster {
```

Comment:
The indentation of the string depends on the closing `"""`. This code block should be indented to align with the rest of the code.